trio/_dtls.py

```diff
@@ -1305,7 +1305,9 @@ async def handler(dtls_channel):
         try:
             self.socket.getsockname()
         except OSError:
-            raise RuntimeError("DTLS socket must be bound before it can serve")
+            raise RuntimeError(
+                "DTLS socket must be bound before it can serve"
+            ) from None
```

Comment:
If you're up for writing a test that triggers this it'd be lovely, otherwise just mark it as `# pragma: no cover` and a comment that a test should be written.

Reply:
Added nocover comment and TODO comment in a5afed7